libs/langchain/langchain/tools/audio_utils.py

```diff
@@ -0,0 +1,18 @@
+import tempfile
+from pathlib import Path
+
+
+def save_audio(audio: bytes) -> str:
```

Comment:
It seems bizzare for me, to have this function and use namedTemporaryFile, why it cannot be user provied path?
On the other hand I generally think that likely langchain does not need to have save/load audio functionality - it is not core and maintenance burden. Can we drop it?
Also it is in wrong place - should be moved to utilities.